### PR TITLE
Feat/intezer improvments

### PR DESCRIPTION
**Pull request by kfirstri**
This is the inside pull request to get your comments :)

### CODE REVIEW

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py [hunk 1]
```diff
@@ -38,164 +50,259 @@
...
-    analysis_ids = argToList(demisto.args().get('analysis_id'))
-    indicator_name = demisto.args().get('indicator_name')
+def get_analysis_metadata_command(intezer_api: IntezerApi, args: Optional[dict]) -> CommandResults:
+    analysis_id = args.get('analysis_id')
+    sub_analysis_id = args.get('sub_analysis_id', 'root')
 
-    for analysis_id in analysis_ids:
-        response = make_analysis_status_request(analysis_id, analysis_type)
-        analysis_result = handle_analysis_result(response)
+    # analysis = get_analysis_by_id(analysis_id, api=intezer_api)
```
Comment (davidt99) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:254]: is it needed?

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py [hunk 2]
```diff
@@ -240,48 +349,60 @@ def enrich_dbot_and_display_endpoint_analysis_results(result, indicator_name=Non
...
-            analyze_by_hash_command()
-        elif demisto.command() == 'intezer-analyze-by-file':
-            analyze_by_uploaded_file_command()
-        elif demisto.command() == 'intezer-get-latest-report':
-            get_latest_result_command()
-        elif demisto.command() == 'intezer-get-analysis-result':
-            check_analysis_status_and_get_results_command()
+
+        intezer_api_key = demisto.getParam('APIKey')
+        intezer_api = IntezerApi(consts.API_VERSION, intezer_api_key, consts.BASE_URL)
```
Comment (davidt99) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:383]: can we allow setting a url as well?

File: Packs/Intezer/pack_metadata.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "name": "Intezer",
     "description": "Malware detection and analysis based on code reuse",
     "support": "partner",
-    "currentVersion": "1.1.1",
+    "currentVersion": "1.1.2",
```
Comment (davidt99) [inline Packs/Intezer/pack_metadata.json:5]: i would change it to 1.2

**Review by davidt99 — APPROVED** [review]
you'll need to add a changelog, see what I did in [here](https://github.com/demisto/content/pull/12486/files#diff-6edc87abce31d6932310b8b9a731cc017dc0f89ce29efed9a3263caa8844ca24)

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py (hunk 1 above)
Comment (kfirstri) [reply Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:254]: nope thanks

File: Packs/Intezer/ReleaseNotes/1_1_2.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+### Integrations
```
Comment (davidt99) [inline Packs/Intezer/ReleaseNotes/1_1_2.md:1]: rename to 1_2.md

**Pull request merged**